content/en/docs/quarks-operator/concepts/ha.md

```diff
@@ -109,20 +111,47 @@ Currently:
 > True if this instance is the first instance of its group.
 ([BOSH jobs](https://bosh.io/docs/jobs/))
 
-The pod that starts first in a statefulet must have the bootstrap flag. It's used to initialize databases and such.
+The pod that starts first in a statefulset must have the bootstrap flag. It's used to initialize databases and such.
 
 In case of an upgrade, the last pod, for example pod-3 has the bootstrap flag. If the pod is restarted, the bootstrap flag has to stay the same.
 
-The startup-ordinal is used to determine this flag.
+The startup-ordinal must be used to determine this flag.
+
+### Calculation
+
+The startup-ordinal is converted into the startup-index:
+If AZs are not used (AZ<2) it's equal to the startup-ordinal, otherwise it's  `(azIndex-1) * 10000 + startup-ordinal`.
+
+
+Now an array of all possible job instances is generated.
```

Comment:
Shouldn't this be `ig instances` instead of `job instances`?

In general a job can appear in multiple instance groups, which can have different instance counts each. A pod corresponds to an ig instance, not a job instance (except when you only talk about instances of this job inside the ig, in which case job and ig kind of become synonymous, but I find that confusing).